### ISSUE TITLE
Display bug with specific extension names. 

### ISSUE THREAD

**Issue by YassiGame** [BlueprintFramework/framework#60]
### Current Behavior

When your blueprint extension has a name with an apostrophe **`'`**, it doesn't display correctly in the extension menu.

### Expected Behavior

Properly displays the extension with its name and logo, with a crisp gradient.
![image](https://github.com/user-attachments/assets/1c31bb5f-0194-4164-bab1-ec58b9a4f4e8)

### Steps to Reproduce

A blueprint add-on with a name containing apostrophes.

![image](https://github.com/user-attachments/assets/7dddbd6a-8ac7-469c-b0a9-305407b1eaeb)
![image](https://github.com/user-attachments/assets/47781a0b-a9ae-4c27-9dc4-a6a3b708998c)

### Blueprint Version

beta-2024-08

### Panel Version

1.11.9

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details and tested my reproduction steps.
- [X] I have checked to my best capabilities and believe this is a bug with the software, and not a configuration issue with my specific system.
- [X] I have made sure to check that this bug is the fault of Blueprint and is not caused by stock Pterodactyl.

**Comment by lattaai1** [comment]
Hello, I've fixed this issue!

This is what I've done:
I have identified the issue with displaying extension names containing apostrophes in the extension menu. The problem was in the 'resources/views/admin/extensions.blade.php' file. I've updated the code to properly escape the extension name when displaying it in the button text.

Here's what I did:

1. I replaced the line that displays the extension name with a properly escaped version using PHP's htmlspecialchars() function.
2. I also made sure to use single quotes around the PHP echo statement to prevent any issues with apostrophes in the extension name.

This change should resolve the issue and allow extension names with apostrophes to display correctly in the extension menu.

I also created pull request: https://github.com/BlueprintFramework/framework/pull/61


> [!CAUTION]  
> Disclaimer: This fix was created by Latta AI and you should never merge before you check the correctness of generated code!

---
This bug was fixed for free by Latta AI - https://latta.ai/ourmission
